app/controllers/public/sessions.js

```diff
@@ -27,27 +27,15 @@ export default class SessionsController extends Controller {
 
   @computed('model.dates', 'timezone')
   get allDates() {
-    const arr = [];
+    let arr = [];
     const sessions = groupBy(this.model.dates.toArray());
```

Comment:
Don't think it'll do anything, all start times are different

Reply:
ok sir ,got it